volatility3/framework/plugins/windows/svcdiff.py

```diff
@@ -61,10 +62,12 @@ def service_diff(
         On Windows 10 version 15063+ 64bit Windows memory samples, walk the services list
         and scan for services then report differences
         """
+        kernel = context.modules[kernel_module_name]
+
         if not symbols.symbol_table_is_64bit(
-            context, symbol_table
+            context, kernel.symbol_table_name
```

Comment:
```suggestion
            context=context, symbol_table=kernel.symbol_table_name
```
No reason for these two to be different